src/app/overview/types.ts

```diff
@@ -37,8 +37,7 @@ export interface GetGlobalBalance {
 
 export interface GetGlobalBalanceSuccess {
     type: GET_GLOBAL_BALANCE_SUCCESS,
-    by: string,
-    globalBalance: number
+    payload: number
```

Comment:
Some payloads have a `?` while others dont. Should they all be mandatory or only some?